clients/src/main/java/org/apache/kafka/clients/admin/Admin.java

```diff
@@ -953,6 +953,14 @@ default ListConsumerGroupOffsetsResult listConsumerGroupOffsets(Map<String, List
      */
     DeleteConsumerGroupsResult deleteConsumerGroups(Collection<String> groupIds, DeleteConsumerGroupsOptions options);
 
+    /**
```

Comment:
Personally, I'd prefer to see these two methods declared with the other share group methods, not mixed in with the consumer group methods.